### PR TITLE
Fixed InputGroup.Addon and InputGroup.Button

### PR DESCRIPTION
**Pull request by psulek**
Fixed InputGroup.Addon and InputGroup.Button sub classes

### CODE REVIEW

**Comment by zhengbli** [comment]
:+1:

File: react-bootstrap/react-bootstrap.d.ts [hunk 1]
```diff
@@ -943,8 +943,8 @@ declare module "react-bootstrap" {
         Addon: typeof InputGroupAddon;
         Button: typeof InputGroupButton;
     }
-    type InputGroup = React.ClassicComponent<InputGroupProps, {}>;
-    var InputGroup: React.ClassicComponentClass<InputGroupProps>;
+    type InputGroup = React.Component<InputGroupProps, {}>;
+    var InputGroup: InputGroupClass;
```
Comment (vsiao) [inline react-bootstrap/react-bootstrap.d.ts]: What about just:
```ts
class InputGroup extends React.Component<InputGroupProps, {}> {
    Addon: ...;
    Button: ...;
}

Comment (psulek) [inline react-bootstrap/react-bootstrap.d.ts]: I'm just follow same pattern as is for **FormControl.Static**

Comment (vsiao) [inline react-bootstrap/react-bootstrap.d.ts]: That's not necessarily the right pattern to follow. `ClassicComponentClass` should only be used for components created with `React.createClass`—[InputGroup does not](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/InputGroup.js).

Comment (psulek) [inline react-bootstrap/react-bootstrap.d.ts]: Good to know. But im just fixup this little part wich was not aligned with other components to be able to use <InputGroup.Addon> in ts. If there is something wrong new issue with proposed changes should be made.

**Pull request merged**